core/src/main/java/nu/yona/server/subscriptions/service/BuddyDto.java

```diff
@@ -58,14 +59,14 @@ public BuddyDto(UserDto user, String personalInvitationMessage, Status sendingSt
 				lastStatusChangeTime);
 	}
 
-	private BuddyDto(UUID id, UserDto user, String personalInvitationMessage, Optional<UUID> userAnonymizedId,
+	private BuddyDto(UUID id, UserDto user, String personalInvitationMessage, Optional<UserAnonymizedDto> userAnonymized,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00107)